pages/about/dataquality.vue

```diff
@@ -1,83 +1,76 @@
 <template>
   <v-layout justify-center column align-content-center>
-    <h1>{{ $t('about.quality.title') }}</h1>
-    <p>{{ $t('about.quality.blurb') }}</p>
-    <h2>{{ $t('gawFull') }}</h2>
-    <woudc-blurb :items="gawBlurb" />
-    <h2>{{ $t('about.quality.sag.title') }}</h2>
-    <woudc-blurb :items="sagBlurb" />
+    <h1 v-text="$t('about.quality.title')" />
+    <p v-text="$t('about.quality.blurb')" />
+    <h2 v-text="$t('gawFull')" />
+    <i18n path="about.quality.gaw-blurb.template" tag="p">
+      <template v-slot:gaw-qa>
+        <a :href="gawURL" target="_blank" v-text="$t('about.quality.gaw-blurb.gaw-qa')" />
+      </template>
+    </i18n>
+    <h2 v-text="$t('about.quality.sag.title')" />
+    <i18n path="about.quality.sag.blurb.template" tag="p">
+      <template v-slot:sop>
+        <nuxt-link :to="localePath('resources-sop')" v-text="$t('about.quality.sag.blurb.sop')" />
+      </template>
+    </i18n>
     <ul>
```

Comment:
Thanks for the suggestion. I tried using `v-list` in that place, but honestly I think it looks waaaay better when the default `ul` styling complements the default HTML on the rest of the page.

Agreed that it should change eventually, but it will be much easier to judge when the rest of the page is given its final style as well. I really think we should leave it be for now, until a major styling pull request comes along.